src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/E2ETestBase.kt

```diff
@@ -49,6 +51,10 @@ class E2ETestBase : MessagingTestBase() {
     sentences = emptyList(),
   )
 
+  internal fun ProbationCase.withChangedMatchDetails(): ProbationCase = this.copy(
+    addresses = this.addresses + ProbationAddress(postcode = randomPostcode()),
+  )
+
```

Comment:
these are a bit easier to work with than `Sentences` - all we actually need here is for the matching fields to change so that `hmpps-person-match` is called